components/eamxx/scripts/test_factory.py

```diff
@@ -81,7 +81,8 @@ def __str__(self):
 class DBG(TestProperty):
 ###############################################################################
 
-    CMAKE_ARGS = [("CMAKE_BUILD_TYPE", "Debug"), ("EKAT_DEFAULT_BFB", "True")]
+    CMAKE_ARGS = [("CMAKE_BUILD_TYPE", "Debug"), ("EKAT_DEFAULT_BFB", "True"),
```

Comment:
Several of the other TestProperty subclasses use the DBG.CMAKE_ARGS list.

SP: `DBG.CMAKE_ARGS + [("SCREAM_DOUBLE_PRECISION", "False")],`
FPE: `DBG.CMAKE_ARGS + [("SCREAM_PACK_SIZE", "1"), ("SCREAM_FPE","True")]`

So this turns on bounds checking for DBG, SP, and FPE. Is that the intent?

Reply:
Ah, no I didn't want to do that. Although, should we? I def don't want it on in the valg build, so I need to edit that. I would argue that bounds can be checked in just one build, since access patterns do not depend on precision or FPE handling... I will make sure other builds don't inherit that.

Reply:
Probably not. You can just explicitly disable it in the SP and FPE tests.

Reply:
VALG does not use DBG.CMAKE_ARGS, so we are fine there.